README.md

```diff
@@ -1,24 +1,171 @@
-# README
+# 📖 [Budget Buddy] <a name="about-project"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+> Budget Buddy is a mobile web application where a user can manage their budget and spending. A User will have a list of transactions associated with a category so that they can track how much money they spent and on what.
 
-Things you may want to cover:
+## 🛠 Built With <a name="built-with"></a>
 
-* Ruby version
+### Tech Stack <a name="tech-stack"></a>
 
-* System dependencies
+<details>
+  <summary>BackEnd</summary>
+  <ul>
+    <li><a href="https://data-sync.onrender.com">Rails</a></li>
+  </ul>
+</details>
 
-* Configuration
+<details>
+  <summary>Test</summary>
+  <ul>
+    <li><a href="https://rspec.com/">RSPEC</a></li>
+  </ul>
+</details>
 
-* Database creation
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
 
-* Database initialization
+<!-- Features -->
 
-* How to run the test suite
+### Key Features <a name="key-features"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+- **[Register and Login]** - Register and Login so the data is private to them.
+- **[Transaction associated with Category]** - Transaction associated with Category.
+- **[Category Records]** - See the money spent on each category.
 
-* Deployment instructions
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-* ...
+<!-- LIVE DEMO -->
+
+## 🚀 Live Version <a name="live-demo"></a>
+
+- [Budget Buddy](budgetism-production.up.railway.app)
+
```

Comment:
- [ ] Your live demo link is broken. Please either use Github pages, Render or Netlify.

Reply:
@emmanuellekamwa I just fixed it before you returned the review